mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -398,9 +398,23 @@ def test_split_files():
 
     assert_allclose(raw_1.buffer_size_sec, 10., atol=1e-2)  # samp rate
     split_fname = op.join(tempdir, 'split_raw.fif')
+    # intended filenames
+    split_fname_elekta_part2 = op.join(tempdir, 'split_raw-1.fif')
+    split_fname_bids_part1 = op.join(tempdir, 'split_raw_part-01_meg.fif')
+    split_fname_bids_part2 = op.join(tempdir, 'split_raw_part-02_meg.fif')
     raw_1.set_annotations(Annotations([2.], [5.5], 'test'))
     raw_1.save(split_fname, buffer_size_sec=1.0, split_size='10MB')
 
+    # check that the filenames match the intended pattern
+    assert op.exists(split_fname_elekta_part2)
+    # check that filenames are being formatted correctly for BIDS
+    raw_1.save(split_fname,
+               buffer_size_sec=1.0, split_size='10MB',
+               split_naming='bids',
+               overwrite=True)
+    assert op.exists(split_fname_bids_part1)
+    assert op.exists(split_fname_bids_part2)
```

Comment:
should also be easy to test that `np.allclose` on the data when reading the `split_fname` (i.e., I/O round-trip)

Reply:
not addressed

Reply:
... oh wait, it is below!